src/lib/xmlparse.rs

```diff
@@ -3675,60 +3726,15 @@ impl<'scf> XML_ParserStruct<'scf> {
                             if cfg!(feature = "mozilla") {
                                 return XML_Error::UNDEFINED_ENTITY;
                             }
-                            current_block_275 = 17939951368883298147;
```

Comment:
In the original C code, this is an early `break` from the enclosing `switch`. Could have used a `bool` variable instead to mark whether you're skipping `doContent_outlined1` or not.

Reply:
Addressed in 1ebb1dc.